### PR TITLE
Inline docs for notnull

### PR DESCRIPTION
**Pull request by shril**
Hi @rxin, take a look at this PR.
This is made in reference to #152 .

Happy to address any issues with my changes.

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -314,8 +314,47 @@ def isnull(self):
...
+        >>> df = ks.DataFrame([(.2, .3), (.0, None), (.6, None), (.2, .1)])
+        ...                    (.2, .1)])
+        >>> df.notnull()
+              0      1
+        0  True   True
+        1  True  False
+        2  True  False
+        3  True   True
+
+        >>> pdf = pd.DataFrame([['ant', 'bee', 'cat'], ['dog', NaN, 'fly']])
```
Comment (rxin) [inline databricks/koalas/frame.py]: change this to
```
df = ks.DataFrame...
```

and then the next line should just be "df"

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -314,8 +314,47 @@ def isnull(self):
...
+        >>> df = ks.from_pandas(pdf)
+              0      1     2
+        0  True   True  True
+        1  True  False  True
+
+        >>> dates = pd.Series(pd.date_range('2013-3-11 21:45:00',
```
Comment (rxin) [inline databricks/koalas/frame.py]: I'd remove this example since it's for Series, not DataFrame

File: databricks/koalas/frame.py (hunk 1 above)
Comment (shril) [reply databricks/koalas/frame.py]: Changed as per requirements.

File: databricks/koalas/frame.py (hunk 2 above)
Comment (shril) [reply databricks/koalas/frame.py]: Removed.

**Comment by rxin** [comment]
@shril the test is failing (look at the Travis CI log). Can you fix that?

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -314,8 +314,36 @@ def isnull(self):
...
+        datatypes, ``None`` or ``NaN`` in objects and ``NaT`` in datetimelike).
+
+
+        See Also
+        --------
+        Dataframe.isnull
+
+        Examples
+        --------
+        >>> df = ks.DataFrame([(.2, .3), (.0, None), (.6, None), (.2, .1)])
```
Comment (HyukjinKwon) [inline databricks/koalas/frame.py]: comma at the end is missing

Comment (shril) [reply databricks/koalas/frame.py]: Fixed.

**Comment by shril** [comment]
Hi @rxin , @HyukjinKwon I have created another branch in which I am pushing the inline documentation of all other functions.
Closing this PR for now.

**Pull request closed without merging**